idc/views.py

```diff
@@ -214,7 +214,7 @@ def quota_page(request):
 
 
 # Data exploration and cohort creation page
-@login_required
+# @login_required
 def explore_data_page(request):
```

Comment:
Just a note, this needs to be restored for master!

Reply:
@s-paquette Sorry. I forgot to add the log required back. Resolved now.